masterlist.yaml

```diff
@@ -21511,6 +21511,92 @@ plugins:
   - name: 'Holds - Falkreath.esp'
     after: [ 'Landscape Fixes For Grass Mods.esp' ]
 
+  ### HS Interior Overhauls ###
+  - name: HSDragonBridge - Four Shields Tavern.esp
+    url:
+      -  link: 'https://www.nexusmods.com/skyrimspecialedition/mods/73017/'
+         name: 'HS DragonBridge - Four Shields Tavern'
```

Comment:
There are two empty spaces inbetween the `-` and `link`, where it only should be one empty space.
The line with the `name` of the link therefore has one empty space too much as well.